tests/unit/AmalgamationInformation.spec.ts

```diff
@@ -50,12 +51,13 @@ for (const test of amalgamationBusinessInfo) {
           entityType: test.entityType,
           tombstone: {
             filingType: FilingTypes.AMALGAMATION_APPLICATION,
-            authRoles: [AuthorizationRoles.STAFF]
+            authorizedActions: []
```

Comment:
Some of the unit tests failed when I left this out (since we're setting the auth role on line 60 so it seems like this isn't needed).

However, I found that some components checked for authorized actions right away, and if we set the tombstone object without the authorizedActions property, trying to de-reference that property causes an exception.

So, I simply initialize it here and that problem doesn't happen.